code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -713,9 +716,9 @@
 
 	if(change_icon_to_default)
 		if(status == BODYPART_ORGANIC)
-			icon = DEFAULT_BODYPART_ICON_ORGANIC
+			icon = icon_greyscale
```

Comment:
What?

Reply:
the define is the greyscale dmi file, which is weird since there's a non greyscale file, so it would make more sense to rename the define to DEFAULT_BODYPART_ICON_GREYSCALE or something similar, I can change it if its wanted.

Reply:
I for some reason thought `DEFAULT_BODYPART_ICON_ORGANIC` wasn't just the greyscale icon path.